SingularityUI/app/components/globalSearch/GlobalSearch.cjsx

```diff
@@ -0,0 +1,52 @@
+React = require 'react'
+
+{ Typeahead } = require 'react-typeahead'
+
+
+GlobalSearch = React.createClass
```

Comment:
Good point.

I did notice that in `components/common` all of the React components are in the `X = React.createClass` syntax.

Is there a reason for that, or should we change them to the `extends` syntax?